src/gleam/string.gleam

```diff
@@ -157,25 +157,34 @@ pub fn slice(from string: String, at_index idx: Int, length len: Int) -> String
   }
 }
 
-// TODO
-// Drop *n* Graphemes from the left side of a
-//
-// ## Examples
-//    > drop_left(from: "The Lone Gunmen", up_to: 2)
-//    "e Lone Gunmen"
-//
-//
-// pub fn drop_left(from string: String, up_to num_graphemes: Int) -> String {}
-// TODO
-// Drop *n* Graphemes from the right side of a
-//
-// ## Examples
-//    > drop_right(from: "Cigarette Smoking Man", up_to: 2)
-//    "Cigarette Smoking M"
-//
-//
-// pub fn drop_right(from string: String, up_to num_graphemes: Int) -> String {}
+/// Drop *n* Graphemes from the left side of a string.
+///
+/// ## Examples
+///    > drop_left(from: "The Lone Gunmen", up_to: 2)
+///    "e Lone Gunmen"
 ///
+///
+pub fn drop_left(from string: String, up_to num_graphemes: Int) -> String {
+  case num_graphemes < 0 || num_graphemes > length(string) {
+    True -> ""
+    False -> erl_slice(string, num_graphemes, length(string) - num_graphemes)
```

Comment:
Can we use `slice` rather than `erl_slice`? I see `erl_slice` as an implementation detail of `slice` which we could later remove without changing any functions other than `slice`.